app/ui-react/syndesis/config.minishift.json

```diff
@@ -40,7 +40,8 @@
     "debugDocumentParsing": false
   },
   "datavirt": {
-    "dvUrl": "/vdb-builder/v1/"
+    "dvUrl": "/vdb-builder/v1/",
+    "enabled": 0
```

Comment:
As those folks said, let's keep this enabled in these files, these are only for dev.